portfolio/src/main/java/com/google/sps/servlets/HelloWorldServlet.java

```diff
@@ -11,7 +11,7 @@
 public class HelloWorldServlet extends HttpServlet {
 
   @Override
-  public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
```

Comment:
In the future, try to avoid whitespace changes like this. (For now, this is fine, but it will be more important in the group project to avoid merge conflicts as much as possible.)